backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -88,8 +103,45 @@ const parseWebhook = async (
   throw new UnexpectedWebhookError('Unexpected webhook body')
 }
 
+const sendPasscodeCreationMessage = async (
+  whatsappId: WhatsAppId,
+  clientId: WhatsAppApiClient
+): Promise<MessageId> => {
+  const templateMessageToSend: WhatsAppTemplateMessageToSend = {
+    recipient: whatsappId,
+    apiClient: clientId,
+    templateName: 'sgc_passcode_generation', // TODO: Un-hardcode this
+    params: [],
+    language: WhatsAppLanguages.english,
+  }
+  const isLocal = config.get('env') === 'development'
```

Comment:
Curious, why do we need this?

Reply:
If you Cmd+Click `sendTemplateMessage`, you'll see that the headers change accordingly.

In dev env, requests have to go through a proxy because your local machine and WhatsApp server are in different network security zones.

In non-dev env, both servers are in the same security zone.

Reply:
I see, thanks for explaining it to me 😄